From fe8e57a36d168c4a2022aaec9df48e0550d9ab26 Mon Sep 17 00:00:00 2001
From: Hood Chatham <roberthoodchatham@gmail.com>
Date: Wed, 22 Dec 2021 21:36:10 -0800
Subject: [PATCH] disable scipy stats mvn

The mvn module uses features from newer versions of Fortran than Fortran 77. 
We can't handle it with f2c.

---
 scipy/stats/_multivariate.py | 1 -
 scipy/stats/kde.py           | 5 ++++-
 scipy/stats/setup.py         | 6 +++---
 3 files changed, 7 insertions(+), 5 deletions(-)

diff --git a/scipy/stats/_multivariate.py b/scipy/stats/_multivariate.py
index 15bd108d8..7d1613bf9 100644
--- a/scipy/stats/_multivariate.py
+++ b/scipy/stats/_multivariate.py
@@ -13,7 +13,6 @@ from scipy.linalg.misc import LinAlgError
 from scipy.linalg.lapack import get_lapack_funcs
 
 from ._discrete_distns import binom
-from . import mvn
 
 __all__ = ['multivariate_normal',
            'matrix_normal',
diff --git a/scipy/stats/kde.py b/scipy/stats/kde.py
index 3fc95931d..287c8e58d 100644
--- a/scipy/stats/kde.py
+++ b/scipy/stats/kde.py
@@ -31,7 +31,10 @@ from numpy import (asarray, atleast_2d, reshape, zeros, newaxis, dot, exp, pi,
 import numpy as np
 
 # Local imports.
-from . import mvn
+try:
+    from . import mvn
+except ImportError:
+    pass
 from ._stats import gaussian_kernel_estimate
 
 
diff --git a/scipy/stats/setup.py b/scipy/stats/setup.py
index 94dc721bb..84dade8e9 100644
--- a/scipy/stats/setup.py
+++ b/scipy/stats/setup.py
@@ -24,9 +24,9 @@ def configuration(parent_package='',top_path=None):
     )
 
     # add mvn module
-    config.add_extension('mvn',
-        sources=['mvn.pyf','mvndst.f'],
-    )
+    # config.add_extension('mvn',
+    #     sources=['mvn.pyf','mvndst.f'],
+    # )
 
     return config
 
-- 
2.25.1

